### DISCUSSION TITLE
Averaged Point Collection

### DISCUSSION THREAD

**Discussion by hadallen** [opengisch/QField#2816] [Q&A]
Hi there, 

I am very interesting in QField and what it can be used for. Does QField have the capability to collect an averaged point? 

An averaged point is where the rover is stationary for point collection, 90 (for example) points are collected (1 point/sec) and the average location of all the points are used to create 1 single point feature. This is a necessary feature for my usage and I would love to include QField in my workflow

**Comment by marioba** [comment]
Hi @hadallen, 
thanks for your question.
At the moment QField does not have this functionality, but we are currently working on it and soon (about a month) it should be released.

**Reply by hadallen** [reply]
Great to hear that it is coming down the line. looking forward to it!
